plugins/autochop.cpp

```diff
@@ -629,13 +688,16 @@ class ViewscreenAutochop : public dfhack_viewscreen
                 if (edit_mode == EDIT_NONE)
                 {
                     x = std::max(x, prev_x + 10);
-                    for (size_t j = 0; j < sizeof(row.skeys)/sizeof(row.skeys[0]); ++j)
+                    for (size_t j = 0; j < sizeof(row.skeys) / sizeof(row.skeys[0]); ++j)
                         OutputString(COLOR_LIGHTGREEN, x, y, DFHack::Screen::getKeyDisplay(row.skeys[j]));
                     OutputString(COLOR_WHITE, x, y, ": Step");
                 }
                 OutputString(COLOR_WHITE, x, y, "", true, left_margin);
             }
             OutputHotkeyString(x, y, "No limit", CUSTOM_SHIFT_N, true, left_margin);
+            OutputToggleString(x, y, "Skip Fruit Trees: ", "f", skip_fruit_trees, true, left_margin);
```

Comment:
It might be better to use CUSTOM_F instead of "f" here (same for the two lines below). I can't think of a reason why anyone would bind CUSTOM_F to anything that isn't "f", but I've seen things like that before.

Reply:
I was trying to follow the style of the other `OutputToggleString` out of an abundance of caution. I'll clean up all the direct letter references here and test them out.

Reply:
Yeah, the ones that take an interface_key are newer than most plugins that use OutputToggleString(), etc.. Thanks!